ci-operator/step-registry/openshift/e2e/azure/cilium/openshift-e2e-azure-cilium-workflow.yaml

```diff
@@ -0,0 +1,34 @@
+workflow:
+  as: openshift-e2e-azure-cilium
+  steps:
+    pre:
+    - chain: ipi-conf-azure
+    - ref: network-conf-disable-cno
+    - ref: cilium-conf
+    - chain: ipi-install
+    test:
+    - ref: openshift-e2e-test
+    env:
+      # 1,2,3) skip NetworkPolicy tests that Cilium doesn't pass yet
+      # 4) Cilium needs a special setting to pass pod-label update test, skip for now
+      #    https://github.com/cilium/cilium/blob/39e09f3ce77bedeac504a55c056da55270fcc6e2/test/kubernetes-test.sh#L3-L7
+      # 5) known Cilium issue
+      #    https://github.com/cilium/cilium/issues/10002
+      # 6) skip maxUnavailable test as openshift-kube-proxy doesn't pass it
+      # 7) skip router idling test as the feature is not supported by Cilium yet (pending investigation)
+      # 8) skip /config/master test (peding investigation)
+      TEST_SKIPS: >-
```

Comment:
@vrutkovs this doesn't seem to work, I thought it did in first iteration and kept using same syntax...

Reply:
hm... maybe I was looking at wrong results.